pkgs/development/python-modules/edlib/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, edlib
+, cython
+, python
+}:
+
+buildPythonPackage {
+  inherit (edlib) pname src meta;
+  version = "1.3.9";
+
+  disabled = pythonOlder "3.6";
+
+  sourceRoot = "source/bindings/python";
+
+  preBuild = ''
+    cp -r ${edlib.src}/edlib .
```

Comment:
If we have the source already, we not cd or symlink?

Reply:
Changed to `ln -s` instead of `cp`, thanks.